noodles-fasta/src/fai.rs

```diff
@@ -1,7 +1,8 @@
 //! FASTA index (FAI) and fields.
 
+mod indexer;
 mod reader;
 mod record;
 mod writer;
 
-pub use self::{reader::Reader, record::Record, writer::Writer};
+pub use self::{indexer::Indexer, reader::Reader, record::Record, writer::Writer};
```

Comment:
```suggestion
pub use self::{reader::Reader, record::Record, writer::Writer};
```

As previously mentioned, `Indexer` should remain private.